src/Symfony/Component/Console/Tests/Helper/TreeStyleTest.php

```diff
@@ -0,0 +1,205 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Console\Tests\Helper;
+
+use PHPUnit\Framework\TestCase;
+use Symfony\Component\Console\Helper\Tree;
+use Symfony\Component\Console\Helper\TreeNode;
+use Symfony\Component\Console\Helper\TreeStyle;
+use Symfony\Component\Console\Output\BufferedOutput;
+use Symfony\Component\Console\Output\OutputInterface;
+
+class TreeStyleTest extends TestCase
+{
+    public function testDefaultStyle()
+    {
+        $output = new BufferedOutput();
+        $tree = self::createTree($output);
+
+        $tree->render();
+
+        $expected = <<<TREE
+root
+├── A
+│   ├── A1
+│   └── A2
+├── B
+│   ├── B1
+│   │   ├── B11
+│   │   └── B12
+│   └── B2
+└── C
+TREE;
+
+        $this->assertSame($expected, trim($output->fetch()));
+    }
+
+    public function testFrameStyle()
+    {
+        $output = new BufferedOutput();
+        $tree = self::createTree($output, TreeStyle::frame());
+
+        $tree->render();
+
+        $expected = <<<TREE
+root
+╠═ A
+╟─ ╠═ A1
```

Comment:
To clean the "frame" style ║ instead of ╟─  if you change it on the other comment